presto-main/src/main/java/com/facebook/presto/execution/SqlQueryManager.java

```diff
@@ -120,10 +125,14 @@
 
     private final SessionPropertyManager sessionPropertyManager;
 
+    private final InternalNodeManager internalNodeManager;
+
     private final Map<Class<? extends Statement>, QueryExecutionFactory<?>> executionFactories;
 
     private final SqlQueryManagerStats stats = new SqlQueryManagerStats();
 
+    private final AtomicBoolean workersStarted = new AtomicBoolean(false);
```

Comment:
The name is not quite accurate. I don't have a good suggestion for it. Maybe acceptQueries? @raghavsethi may have a better suggestion.